recipes/libcf/build.sh

```diff
@@ -0,0 +1,15 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
```

Comment:
Don't worry about that in `conda-forge`. In a normal build there should be nothing in there. However, we want the scripts to be generic enough so other people with different build systems can inject whatever they need there. (We even have packages that do that in `conda-forge` to ensure some `OS X` specific variables for `clang` are used.)

So if a user is modifying that they should know what they are doing.